### PR TITLE
Census 3 changes

### LINKED ISSUE #300: Improve "Token Voting" census
For the token based, we need to add a new drop-down to select the network (Ethereum, Polygon, Optimism...). 

The token dropdown will use the "select" dropdown with group & search functionalities: 
- [x] Group tokens by type (ERC20, ERC721, POAP) and change the group name to "Tokens", "NFTs", "POAPs"
- [x] Add a search functionality and search for "Name" or "Token address"
- [x] Add a new item in the bottom: "Add Custom Token". When clicked, open the Typeform link
- [x] If the user search and "No results found", also add the "Add Custom Token" in below it with the same functionality

We should add the icon of the network ( we can use: [CryptoMeta](https://github.com/ViewBlock/cryptometa) ) that offers this URL to generate the network icon: [https://meta.viewblock.io/{key}/logo](https://meta.viewblock.io/%7Bkey%7D/logo)). 

We should also add the token icon when selected, using the token address and this repo: [Token Icons](https://github.com/albertocevallos/token-icons/tree/main/assets/blockchains/ethereum/assets). If the token don't have a valid logo, use the current approach.

Also, add the "Holders" number as info inside the token card.

image

image

Tasks:
- [x] : New Network dropdown
- [x]   Improve token dropdown with name/address search, "add custom token" and "no result found"
- [x] : Add network icon
- [x] : Add token icon
- [x] : Add the "Holders" information inside the token card.

### PR DESCRIPTION
**Pull request by elboletaire**
closes #300

### CODE REVIEW

File: public/assets/polygon-matic.jpg
Comment (elboletaire) [inline public/assets/polygon-matic.jpg:1]: Compared to the ethereum one, this one's too big.

File: src/theme/tokenSelectStyles.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { ChakraStylesConfig } from 'chakra-react-select'
+
+export const customStyles: ChakraStylesConfig = {
```
Comment (elboletaire) [inline src/theme/tokenSelectStyles.ts:3]: Find a better name than `customStyles`.

**Review by elboletaire — COMMENTED** [review]
Note we cannot approve this until a new  version of the SDK is published (and we remove the custom version package stored in the repository).

**Review by gerouvi — APPROVED** [review]

**Comment by elboletaire** [comment]
Note this has been closed because it was directly added via PR #399 (in ae59a930595d446554c326f8d7660fd7d270f79c)

**Pull request closed without merging**